platform/engineering/redirect-implementation-strategy.md

```diff
@@ -1,130 +1,220 @@
+
+Consider if the redirect should be a `catchAll`. Example [here](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/proxy-rewrite/redirects/crossDomainRedirects.json#L1174)
 
 _What team is responsible?_
-- Work is routed from VSP to Public Websites team for implementation (label ticket with `⭐️ Public Websites`). Public Websites team to pull in VSP Ops as needed. Requesting team (whether that is Public Websites or another VFS team) is responsible for communication with VA stakeholders as needed.
+- Public Websites, in collaboration with whatever veteran-facing team is requesting the redirect.  Requesting team is responsible for communication with VA stakeholders as needed.
+
+_Any other notes?_
+- For the majority of page-level redirects from subdomains (subdomain.va.gov), this is not feasible in the short term without TeamSite/proxy-rewrite JS because it requires access to site hosting and relationships with technical stakeholders on the relevant teams. These contacts should be actively sought out via OCTODE.
+- Client-side redirects are not ideal, as it requires JavaScript to be enabled & running correctly in order for a user to be redirected properly. It also depends on the legacy page staying alive. Should be considered temporary solution.
+  - In November 2019, Public Websites team worked with TeamSite stakeholder Sandy Tadeo to test archiving a TeamSite page that had a client side redirect on it. The result was that sometimes the page would load a 404 for the user before redirecting. As a result of this test, the team determined **not** to move forward with archiving redirected TeamSite pages. Instead, Public Websites will implement the client side redirect, with additional ask of VA stakeholder to: 
+    - A) Replace their page content with a 'redirect' message and 
+    - B) Update their print materials and nav links with the new URL
 
 _Any other notes_
-- This is not ideal, as it depends on our JavaScript running correctly in order for a user to be redirected properly. It also depends on the legacy page staying live. Should be considered temporary solution.
-- In November 2019, Public Websites team worked with TeamSite stakeholder Sandy Tadeo to test archiving a TeamSite page that had a client side redirect on it. The result was that sometimes the page would load a 404 for the user before redirecting. As a result of this test, the team determined **not** to move forward with archiving redirected TeamSite pages. Instead, Public Websites will implement the client side redirect, with additional ask of VA stakeholder to: A) Replace their page content with a 'redirect' message and B) Update their print materials and nav links with the new URL
-- Level of difficulty: low
+- Level of difficulty: low if proxy-rewrite is in play; high if not (dependent on VA stakeholders)
+
+
+### 5) Redirects from subdomains (subdomain.va.gov) to subdomains (subdomain.va.gov)
```

Comment:
Is there another case where this is doable, or are we tabling this functionality for now?

Reply:
Updated the language here to be more clear about what will / won't happen.